cvmfs/sync_mediator.cc

```diff
@@ -39,7 +44,8 @@ SyncMediator::SyncMediator(catalog::WritableCatalogManager *catalog_manager,
   union_engine_(NULL),
   handle_hardlinks_(false),
   params_(params),
-  changed_items_(0)
+  changed_items_(0),
```

Comment:
Do we still need `changed_items_` here since we have it already in the diff reporter?